operators/understanding/olm-packaging-format.adoc

```diff
@@ -62,3 +56,26 @@ For instructions about creating file-based catalogs by using the `opm` CLI, see
 For reference documentation about the `opm` CLI commands related to managing file-based catalogs, see xref:../../cli_reference/opm/cli-opm-ref.adoc#cli-opm-ref[CLI tools].
 
 include::modules/olm-fb-catalogs-automation.adoc[leveloffset=+2]
+
+include::modules/olm-rukpak-about.adoc[leveloffset=+1]
+
+The following sections further detail the individual components of RukPak.
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../operators/admin/olm-managing-po.adoc#olm-managing-po[Managing platform Operators]
+* xref:../../operators/admin/olm-managing-po.adoc#olm-po-techpreview_olm-managing-po[Technology Preview restrictions for platform Operators]
+
+include::modules/olm-rukpak-bundle.adoc[leveloffset=+2]
+include::modules/olm-rukpak-bundle-immutability.adoc[leveloffset=+3]
+include::modules/olm-rukpak-plain-bundle.adoc[leveloffset=+3]
+include::modules/olm-rukpak-registry-bundle.adoc[leveloffset=+3]
+[role="_additional-resources"]
```

Comment:
It is probably totally fine, but it always weirds me out not having a blank line above things like this :slightly_smiling_face: , because sometimes things get messed up if you don't. It's fine to leave as is, but maybe consider the space just to make sure there aren't any issues.